Runtime/Core/BucketManagement/BucketStore.cs

```diff
@@ -384,5 +373,17 @@ public GetBucketPoliciesResponse GetBucketPolicies(GetBucketPoliciesRequest requ
 
             return Response.Fail(response);
         }
+        
+        public virtual string GetBucketPolicyPath()
+        {
+            string packageName = "com.amazonaws.gamelift";
```

Comment:
This should be pulled from Paths.cs `PackageName` instead!

Reply:
Apart from this, it all looks great to me!

Reply:
Paths.cs is in another assembly

Reply:
Ok so the problem is that the resource file itself is present in the Editor assembly but is being referenced in the Runtime assembly. Is there a reason why the json shouldn't live in the Runtime assembly? Its never used in the Editor assembly so shouldn't reside there

Reply:
Moved policy json

Reply:
No

Reply:
From a quick search it seems you should be able to do this using `Directory.GetCurrentDirectory()` i believe. There are some other references in our codebase as well. Is that applicable?